src/UserCard.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles({
+  root: {
+    maxWidth: 128
+  }
+});
+
+// component: semantic, variant: style
+export default function UserCard({ userId }) {
+  const [user, setUser] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    (async function() {
+      const res = await fetch(`https://reqres.in/api/users/${userId}`);
+      const newUser = await res.json();
+      setUser(newUser.data);
+      setIsLoading(false);
+    })();
+  }, [userId]);
```

Comment:
👍 Nice work including the dependency here. This is an easy thing to forget and can cause some weird bugs.